### ISSUE TITLE
Arabic RTL or reshaping issue in AnkiDroid

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#1582]
Originally reported on Google Code with ID 679

```
What steps will reproduce the problem?
1.Input arabic text on computer and load anki deck on android 2.2/
2.
3.

What is the expected output? What do you see instead?
Desired output is arabic text RTL.  Text that it output is normal.  

What version of AnkiDroid are you using? On what version of Android?
0.6
0.7 alpha 4
or 0.7 beta 6

Please provide any additional information below.
This is a known android issue but I saw a previous post that listed a additional script
that was added to the alpha version and now works.  I loaded this but the test was
still LTR instead of RTL.  
https://github.com/simar0at/Anki-Android/commit/1f663a64799012f5557b96b7e894b2a050eb09a4

Further info, my phone doesn't do RTL also but I know that the java work around should
correct this internal to the program.  

I can upload a deck to use as an example if desired
```

Reported by `jhtravis@quikermail.com` on 2011-07-10 17:06:51

**Comment by dejj** [comment]
I am using AnkiDroid 2.7, and I am having trouble with RTL display of Arabic text. The display of opening and closing brackets is confused, similar to https://github.com/ankidroid/Anki-Android/issues/3628 .
The text displays correctly in the card editor and deck editor, only the card view is concerned.
I do combine LTR and RTL text.

**Comment by eginhard** [comment]
Did you follow the [instructions for using RTL text](http://ankisrs.net/docs/manual.html#rtl-(right-to-left)-text) from the manual that are also linked in that bug?

**Comment by dejj** [comment]
I did adjust the template, although this deck-wide setting should not be hidden away inside the card editor. The resulting change wasn't useful:
text in editor: قريب (من) qaríb (min)
display without dir=rtl:  ؜ (قريب (من qaríb (min)
display with dir=rtl: (qaríb (min (من) قريب

I just noticed how difficult it is to input the wrong variants even in the browser. The input method moves the brackets around and changes them from closing to opening on its own. I have no idea how to even predict the behaviour, so debugging it would be next to impossible for me.

In the app, I assume the deck-view is a plaintext display, while the card-view is an HTML display.
Is there a way to force plaintext display for the card-view, perhaps?
